src/libkomb.py

```diff
@@ -427,7 +427,7 @@ def komap_mapbox(options, style):
                 )
 
                 mapbox_style_layer["layout"]["text-field"] = to_mapbox_expression(
-                    {z: v.to_mapbox_expression() for z, v in st.get("text").items()}
+                    {z: v.to_mapbox_expression(lambda tag: tag if tag != "name" else "name:%s" % (options.locale)) for z, v in st.get("text").items()}
```

Comment:
coalesce?

Reply:
coalesce will work too but just replace `name` with `name:[lang]` seems simpler for me

Reply:
without coalesce you will have to put copies of name into possibly wrong language fields?